### DISCUSSION TITLE
Error observed during checkbox column creation [Apr 10]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5496] [Discord Threads]
> 🙋‍♂️ **dev_dipper**  
#### what am I missing? thanks  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1083796421133750393/image.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### Can you help with project information? You can click on project menu on left top & `copy project info`  
#  
> 🙋‍♂️ **dev_dipper**  
#### Node: **v16.19.1**   
#### Arch: **x64**   
#### Platform: **linux**   
#### Docker: **true**   
#### RootDB: **pg**   
#### PackageVersion: **0.105.3**  
#  
> 🙋‍♂️ **dev_dipper**  
#### thanks <@857998597474484295> 🙂  
#  
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### worked fine for me. tried changing default value as well  
#  
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### help us create a github issue with relevant details  
#  
> 🙋‍♂️ **dev_dipper**  
#### hey <@857998597474484295> , found the error: cleared the cells and it works, one of them must have contained a space 🙂  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1083796421410557972)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
